### PR TITLE
Remove pyrmont testnet

### PR DESCRIPTION
**Pull request by tqpcharlie**
## PR Description
This will remove the pyrmont testnet when it will get shut down.

## Documentation

- [x] I thought about documentation and added the `documentation` label to this PR if updates are required.

## Changelog

- [x] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

**Comment by benjaminion** [comment]
Thank you for the PR - it is much appreciated!

It will likely be a few weeks/months before we can merge this. Although Pyrmont is deprecated for new users, client teams (like ourselves) are continuing to use the network for experimentation and testing of some extreme scenarios. Thus we need to keep Pyrmont network support available for a little while longer.

File: ethereum/spec/src/main/java/tech/pegasys/teku/spec/networks/Eth2Network.java [hunk 1]
```diff
@@ -19,7 +19,6 @@
 public enum Eth2Network {
   // Live networks
   MAINNET,
-  PYRMONT,
```
Comment (benjaminion) [inline ethereum/spec/src/main/java/tech/pegasys/teku/spec/networks/Eth2Network.java:22]: Also need to remove [line 22](https://github.com/ConsenSys/teku/pull/4405/files#R22) of _Eth2NetworkConfiguration.java_ else compilation fails:

```
import static tech.pegasys.teku.spec.networks.Eth2Network.PYRMONT;
```

Comment (benjaminion) [reply ethereum/spec/src/main/java/tech/pegasys/teku/spec/networks/Eth2Network.java:22]: Running `./gradlew spotlessApply` will catch errors like this and also ensure that the code formatting conforms to our specification (building will fail if it doesn't).

Running `./gradlew build` locally compiles the code and runs the unit tests. This is well worth doing to catch issues before committing.

**Comment by ajsutton** [comment]
I'm going to close this PR for now. At this stage we're likely to keep the Pyrmont config in Teku for at least another couple of months so that if anyone is trying to connect they still can, even though the network is gradually fading away as validators exit.  There's already quite a few conflicts because of other changes going on so it's not going to be easy to update this PR and no benefit to keeping it open until then.

I do appreciate the effort you've put in though.

**Comment by ghost** [comment]
I've continued and fixed this PR in #4970.

**Pull request closed without merging**